<launch>
    <node name="sensor_node" pkg="wrist_node" type="sensor_node.py">
        <param name="w1_addr" value="0" type="int" />
        <param name="w2_addr" value="1" type="int" />
        <param name="sensor_port" value="/dev/ttyUSB1" type="string" />
        <param name="baud_rate" value="1000000" type="int" />
    </node>
</launch>
